operator/internal/manifests/openshift/configure.go

```diff
@@ -73,6 +73,19 @@ func ConfigureGatewayDeployment(
 		return kverrors.Wrap(err, "failed to merge sidecar container spec ")
 	}
 
+	if mode == lokiv1.OpenshiftLogging {
+		// enable extraction of namespace selector
+		for i, c := range d.Spec.Template.Spec.Containers {
+			if c.Name != "gateway" {
+				continue
+			}
+
+			d.Spec.Template.Spec.Containers[i].Args = append(d.Spec.Template.Spec.Containers[i].Args,
+				"--logs.extract-selector-labels=kubernetes_namespace_name",
```

Comment:
```suggestion
				fmt.Sprintf("--logs.extract-selector-labels=%s", opaDefaultLabelMatcher),
```
I suggest to replace the hardcoded name with the `opaDefaultLabelMatcher` constant